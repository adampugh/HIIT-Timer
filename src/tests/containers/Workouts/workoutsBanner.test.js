import React from "react";
import { shallow } from "enzyme"; 

import WorkoutsBanner from "../../../containers/Workouts/workoutsBanner";

test("should render workoutsBanner correctly", () => {
    const wrapper = shallow(<WorkoutsBanner />);
    expect(wrapper).toMatchSnapshot();
});