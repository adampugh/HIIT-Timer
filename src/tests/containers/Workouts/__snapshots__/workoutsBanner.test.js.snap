// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render workoutsBanner correctly 1`] = `
<div
  className="bannerText"
>
  <div
    className="bannerText__container"
  >
    <img
      alt="arm icon"
      className="workoutsBanner__image"
      src="arm-icon.png"
    />
    <h2>
      Add a new workout and start today! Click the + button below to add a new workout.
    </h2>
  </div>
</div>
`;
